addons/account/models/account_move.py

```diff
@@ -3499,6 +3499,10 @@ def _post(self, soft=True):
                 customer_count[invoice.partner_id] += 1
             elif invoice.is_purchase_document():
                 supplier_count[invoice.partner_id] += 1
+            elif invoice.move_type == 'entry':
+                sale_amls = invoice.line_ids.filtered(lambda line: line.debit and line.partner_id and line.account_id.account_type == 'asset_receivable')
+                for partner in sale_amls.mapped('partner_id'):
+                    customer_count[partner] += 1
```

Comment:
why only for customer and not for vendor?
why checking on debit if a refund should also count as a sale too?

```suggestion
                sale_amls = invoice.line_ids.filtered(lambda line: line.partner_id and line.account_type == 'asset_receivable')
                for partner in sale_amls.partner_id:
                    customer_count[partner] += 1
                purchase_amls = invoice.line_ids.filtered(lambda line: line.partner_id and line.account_type == 'liability_payable')
                for partner in sale_amls.partner_id:
                    supplier_count[partner] += 1
```

Reply:
this is only for misc operations, so there is no refund or incoice here, and I tried only covering the spec of task by covering only customers because it's related to follow up reports

Reply:
@moerradi If you handle customer partner for misc operations, you need to do the same for suppliers too. Since a misc operation could be anything either a refund or a sale or anything, just make it generic for everything then

Reply:
Aright !